server/logger.js

```diff
@@ -37,7 +37,7 @@ const initLogger = () => {
       new winston.transports.File({
         filename: path.join(
           __dirname,
-          '../../../optimize/wazuh-logs/wazuhapp.log'
+          '../../../optimize/wazuh/logs/wazuhapp.log'
```

Comment:
The log folder path is repeated a lot, maybe you can save it to a constant. 
```js
const logsPath = '../../../optimize/wazuh/logs';

path.join(__dirname, `${logsPath}/wazuhapp.log`);
```

Reply:
Here if you are right, please change `../../optimize/wazuh/logs`  with a constant.